pkg/handler/handler.go

```diff
@@ -37,9 +39,18 @@ import (
 	"k8s.io/klog/v2"
 )
 
+// This metric is used to count the number of pods that are mutated for either sts web identity or container credentials
+var webhookPodCount = prometheus.NewCounterVec(
```

Comment:
Similar comment as below, can you move this to be with the other metrics in https://github.com/aws/amazon-eks-pod-identity-webhook/blob/master/pkg/handler/middleware.go